tasking/pulp/tasking/tasks.py

```diff
@@ -149,7 +149,7 @@ class UserFacingTask(PulpTask):
     # this tells celery to not automatically log tracebacks for these exceptions
     throws = (PulpException,)
 
-    def apply_async_with_reservation(self, resource_type, resource_id, *args, **kwargs):
+    def apply_async_with_reservation(self, resource_type, resource_id, *async_args, **async_kwargs):
```

Comment:
I think this needs to be `args=None, kwargs=None` instead of `*async_args, **async_kwargs`. It needs to match the function signature of celery's apply_async. http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async